examples/topics/stats/pyramid.py

```diff
@@ -0,0 +1,45 @@
+''' A pyramid plot showing the age distribution of the male and female Titanic passengers using the `Bokeh` hbar glyph.
```

Comment:
> age distribution of the male and female Titanic passengers

Perhaps this should be the title of the plot instead of the existing one.

Reply:
Alright.